tests/rbd_mirror/test_mirror_promote.py

```diff
@@ -0,0 +1,54 @@
+from tests.rbd.rbd_utils import Rbd
+from tests.rbd_mirror import rbd_mirror_utils as rbdmirror
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(**kw):
+    """To promote secondary cluster as primary after primary cluster fails,
+    which could be used for Regional DR scenario.
```

Comment:
Can we generalize doc ?
(Remove Regional DR scenario and add mirroring specific words)

Reply:
done

Reply:
Module has been removed, marking as resolved.